test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -241,12 +241,14 @@ def test_framework_version_cpu(image):
         else:
             if "neuron" in image:
                 assert tag_framework_version in output.stdout.strip()
-            if all(_string in image for _string in ["pytorch", "habana", "synapseai1.4.1"]):
+            if (all(_string in image for _string in ["pytorch", "habana", "synapseai1.4.1"])
```

Comment:
Why is `synapseai1.4.1"` present in `all`?

Reply:
fixed